src/main/java/roomescape/member/MemberRepository.java

```diff
@@ -0,0 +1,42 @@
+package roomescape.member;
+
+import org.springframework.stereotype.Repository;
+
+import java.util.Map;
+
+@Repository
+public class MemberRepository {
+
+    private static final Member member1 = new Member("제이슨", "json@email.com", "1234");
+    private static final Member member2 = new Member("심슨", "simson@email.com", "1234");
+    private static final Map<Long, Member> MemberStorage = Map.of(
+            member1.getId(), member1,
+            member2.getId(), member2);
+
+    public boolean isExistByEmail(Member member){
+        return MemberStorage.values().stream().anyMatch(
+                m -> m.getEmail().equals(member.getEmail())
```

Comment:
```suggestion
                member -> m.getEmail().equals(member.getEmail())
```

이왕이면 줄임말 대신 명확한 이름을 사용하면 좋을 것 같네요!